Users/Tutorial.rst

```diff
@@ -158,6 +158,12 @@ current directory. This makes it easy to specify once for your project
 what is checked with which bears and make it available to all
 contributors.
 
+The line ``[cli]`` implies that everything below belongs to the special
```

Comment:
still feel like cli should not be brought up first here, explain how to define sections and their meaning using any arbitrary name and then explain all about cli that its the command line arguments from previous coala run. 

**If you are sticking to the issue then better just do that and not add additional stuff or rearrange matter,**

**but if you are reframing many things then I feel like you need to:**

1. explain what is a section, and how to define it
2. what is this [cli] section, that has been formed automatically and why we shouldn't make one on our own
3. what is section inheritance

Reply:
or if you are removing section inheritance since that is properly explained in another doc, then create an issue for it status/blocked by this issue since explaining here will require you to repeat the things over there or what is already here is encouraging deprecated behaviour

Reply:
please ping a maintainer if you disagree, this is just my perspective 😉

Reply:
I like all of the above suggestions and I think that we should file a new issue -- *Improve the __Getting Started__ tutorial* -- to incorporate all those changes. :-) 
@jayvdb what do you think?